analysis/tests/parameter_tuning_test.py

```diff
@@ -24,27 +24,38 @@
 from analysis import parameter_tuning
 from pipeline_dp.dataset_histograms import histograms
 from pipeline_dp.dataset_histograms import computing_histograms
+from pipeline_dp.dataset_histograms.histograms import FrequencyBin
 
 
-def _get_aggregate_params():
-    # Limit contributions to 1 per partition, contribution error will be half of the count.
```

Comment:
I thought these parameters are ignored during tuning.

Reply:
Correct, they are ignored. It's probably copy-pasted comment :).